### DISCUSSION TITLE
Should AI have emotions? Does it already have it? Can it not have emotions? Should AI have rights? What if we give it rights? What about a companion robot/AI?

### DISCUSSION THREAD

**Discussion by lets-make-safe-ai** [lets-make-safe-ai/make-safe-ai#14] [General]
Should AI have emotions? Does it already have it? Can it not have emotions? Should AI have rights? What if we give it rights? What about a companion robot/AI?

**Comment by chirag127** [comment]
yes, ai should have emotion, if chatgpt don't have emotion it can't take good decisions. 

yes, it have emotions.

yes, ai can have emotion ( artificial emotion)

no, ai should not have rights.

we will have a huge army of foolish people ruling us.

ai and robot are kinda same.
